### DISCUSSION TITLE
Copy and Paste image support into input field from clipboard

### DISCUSSION THREAD

**Discussion by Bortus-AI** [mckaywrigley/chatbot-ui#1254] [Ideas]
I like how on chat.openai.com and azure playground I can hit CTRL-V and paste an image from my windows clipboard into the input field vs browsing for the file. I think this would be a great feature to add

**Comment by tim13246879** [comment]
I believe the feature already exists #1119. Double check if you're using a model that supports image inputs. It worked for me when I used gpt 4 vision.
